Box_intersection_d/include/CGAL/Box_intersection_d/segment_tree.h

```diff
@@ -275,9 +275,8 @@ split_points( RandomAccessIter begin, RandomAccessIter end,
 
 
 #if CGAL_BOX_INTERSECTION_DEBUG
- static int level = -1;
  #define CGAL_BOX_INTERSECTION_DUMP(msg) { \
-   for( unsigned int i = level; i; --i ) \
+   for( unsigned int i = -1; i; --i ) \
```

Comment:
This patch completely modifies the loop. As such the loop will loop from `-1`, that is `UINTMAX` as a `unsigned`, decreasing to to `0`, whereas before the patch it was looping from `level` to `0.